tests/testthat/test-error.R

```diff
@@ -15,4 +15,17 @@ test_that("returns the correct linting", {
     function() {
     b",
     rex("unexpected end of input"), (function(...) NULL))
+
+  expect_lint("x=",
```

Comment:
If we modify the default linters, these tests might need rewriting. Rather than using all of the default linters in these tests, could you use a single specific linter, that *shouldn't* throw on the code, for example `equals_na_linter`. That way, only the parsing errors should be caught.

For example,
`expect_lint("x=", rex("unexpected end of input"), equals_na_linter)`

Also, could you add a couple of additional tests:
- `{x = }`
- and `x = ;`

should both throw parsing errors, but should throw no other linting errors when using `equals_na_linter`

Reply:
I was thinking about this too. Thanks for your nice suggestions!